index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Masonry Layout</title>
+</head>
+<body>
+    <div class="container">
+        <div class="box">
+            <div class="content">
+                <img src="img/1.png" alt="">
```

Comment:
Provide ```alt``` value in case the page doesn't render correctly from the user side.

Reply:
ooh let me give value to alt attribute

Reply:
it is been a month I still don't see changes!!

Reply:
sorry!!
done now...